src/main/java/ch/njol/skript/Skript.java

```diff
@@ -489,6 +489,16 @@ public void onEnable() {
 			classLoadError = e;
 		}
 
+		// Warn about pausing
+		if (Skript.methodExists(Server.class, "getPauseWhenEmptyTime")) {
+			int pauseThreshold = getServer().getPauseWhenEmptyTime();
+			if (pauseThreshold > -1) {
+				Skript.warning("Server pausing is enabled!");
+				Skript.warning("Scripts that interact with the world or entities will not work when the server is paused and may crash your server.");
```

Comment:
"will not" doesn't seem accurate to me.
Since there are many different ways to interact with a World/Entity, a blanket "will not" doesn't seem appropriate.
Maybe instead:
```suggestion
				Skript.warning("Scripts that interact with the world or entities may not work as intended when the server is paused and may crash your server.");
```